spec/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+## Specification PR template
+When creating a specification project release review, create PRs with the content defined as follows.
+
+Include the following in the PR:
+- [ ] A directory in the form <project-name>/x.y where <project-name> is the specificaiton project name
+(micropfile-config, microprofile-health, ...) and x.y is the release major.minor version, and the directory contains the following.
+- [ ] Specification PDF in the form of microprofile-project-spec-x.y.pdf where
```

Comment:
See my comment above. In this case, the spec file name <project-name>-spec.pdf and <project-name>-spec.html since the version was contained in the directory already.

Reply:
See updated simplifications